packages/slate/src/models/range.js

```diff
@@ -674,6 +674,12 @@ class Range extends Record(DEFAULTS) {
     const range = this
     let { anchorKey, anchorOffset, focusKey, focusOffset, isBackward } = range
 
+    const anchorOffsetType = typeof anchorOffset
+    const focusOffsetType = typeof focusOffset
+    if (anchorOffsetType != 'number' || focusOffsetType != 'number') {
+      logger.warn('The range offsets should be numbers, but they were of type', anchorOffsetType, 'and', focusOffsetType)
```

Comment:
Can you make this a `` style string, with the type variables interpolated? And add `"` around each of them, that way it'll show up as:

> Range offsets should be numbers, but they were of type "string" and "string".

Which will hopefully be easier to read quickly. And add a period, thanks!

Reply:
done :) I appreciate the call for consistency!